packages/usa-form/src/usa-form.stories.js

```diff
@@ -54,3 +54,4 @@ DisabledFormElements.argTypes = {
     table: { disable: false },
   },
 };
+DisabledFormElements.title = "TESTING";
```

Comment:
Is this needed?

Reply:
Removed in https://github.com/uswds/uswds/pull/5871/commits/4175e364be9453051d40e882cdc9fe73626a42f1